### PR TITLE
Change readme language to swedish

### PR DESCRIPTION
**Pull request by bwiberg**

### CODE REVIEW

**Comment by jon-grangien** [comment]
GOOD work.

**Comment by dragonslayern** [comment]
hahahah god

**Comment by jon-grangien** [comment]
ship it

**Comment by joekay** [comment]
gg

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # TowerVR
 
-## Specification
+## Specifikation
```
Comment (dragonslayern) [inline README.md:4]: Nice joob!

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
 Features
-* Tower stacking
+* Bygga torn
```
Comment (jon-grangien) [inline README.md:6]: STUPID descritpion!!!!

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # TowerVR
```
Comment (adamalsegard) [inline README.md:2]: FELSTAVNING!!

File: README.md (hunk 2 above)
Comment (bwiberg) [inline README.md:6]: Whoops, closing pull request then :/

**Comment by bwiberg** [comment]
Sadface :(

**Pull request closed without merging**